controllers/api/eventRoutes.js

```diff
@@ -1,38 +1,176 @@
 const router = require('express').Router();
-const { Event } = require('../../models');
+const { Event, User, Task } = require('../../models');
+const sequelize = require('../../config/connection');
 const withAuth = require('../../utils/auth');
 
-router.post('/', withAuth, async (req, res) => {
-  try {
-    const newEvent = await Event.create({
-      ...req.body,
-      user_id: req.session.user_id,
+// get all users
+router.get('/', (req, res) => {
+  console.log('======================');
+  Event.findAll({
+      attributes: [
+        'id',
+        'name',
+        'event_date',
+        'event_address',
+        'event_owner'
+      ],
+    order: [['created_at', 'DESC']],
```

Comment:
There isn't a "created_at" column in the event table so you won't be able to order by that attribute. You could do this:
order: [['event_date', 'DESC']],

This will give the events in order of when the event will occur.